src/adapter/z-stack/adapter/startZnp.ts

```diff
@@ -35,7 +35,7 @@ const EndpointDefaults: {
 const Endpoints = [
     {...EndpointDefaults, endpoint: 1, appprofid: 0x0104},
     {...EndpointDefaults, endpoint: 2, appprofid: 0x0101},
-    {...EndpointDefaults, endpoint: 3, appprofid: 0x0105},
+    {...EndpointDefaults, endpoint: 3, appprofid: 0x0104, appnuminclusters: 1, appinclusterlist: [Zcl.Utils.getCluster('hvacThermostat').ID]},
```

Comment:
This change potentially breaks other devices, what adapter are you using? (as instead the FW should be modified)

Reply:
I'm on CC2531, I tried with ZStack 3.0.x and Z-Stack Home 1.2.

Reply:
Instead of changing this line, can you try changing `{...EndpointDefaults, endpoint: 1, appprofid: 0x0104},` to `{...EndpointDefaults, endpoint: 1, appprofid: 0x0104, appnuminclusters: 1, appinclusterlist: [Zcl.Utils.getCluster('hvacThermostat').ID},` reflash your stick and see if that also works?

Reply:
Tested it with Z-stack home and it odes not work. 
No reads happen.

Reply:
Can we maybe have a configuration switch?

Reply:
Can you try with this firmware (it's the zstack home 1.2 source routing firmware with some modification). Also make sure to discard your changes.

[CC2531ZNP-Prod.zip](https://github.com/Koenkk/zigbee-herdsman/files/6487568/CC2531ZNP-Prod.zip)

Reply:
Tested with this firmware and reverted startZnp.js and it does work!

Reply:
Perfect!